src/bin/bootstrap/src/main.rs

```diff
@@ -10,55 +28,38 @@ fn find_init_name(name: &str) -> Option<ObjID> {
     None
 }
 
-use dynlink::{
-    library::{Library, LibraryLoader},
-    symbol::LookupFlags,
-    DynlinkError,
-};
-use tracing::Level;
-use tracing_subscriber::FmtSubscriber;
-use twizzler_abi::{
-    object::ObjID,
-    syscall::{sys_object_create, ObjectSource},
-};
-
-fn start_runtime(runtime_monitor: ObjID, runtime_library: ObjID, libstd: ObjID) {
+fn start_runtime(runtime_monitor: ObjID, _runtime_library: ObjID) -> ! {
     let ctx = dynlink::context::Context::default();
     let monitor_compartment = ctx.add_compartment("monitor").unwrap();
 
+    // TODO: we should not hardcode these names, and make it flexible as to what is loaded in bootstrap.
     let mon_library = Library::new(
         Object::<u8>::init_id(runtime_monitor, Protections::READ, ObjectInitFlags::empty())
             .unwrap(),
         "monitor",
     );
 
+    /*
     let rt_library = Library::new(
         Object::<u8>::init_id(runtime_library, Protections::READ, ObjectInitFlags::empty())
             .unwrap(),
         "runtime",
     );
-
-    let _libstd_library = Library::new(
-        Object::<u8>::init_id(libstd, Protections::READ, ObjectInitFlags::empty()).unwrap(),
-        "libstd",
-    );
+    */
 
     let mut loader = Loader {};
     let monitor = ctx
         .add_library(&monitor_compartment, mon_library, &mut loader)
         .unwrap();
-    let runtime = ctx
+    /*let runtime = ctx
```

Comment:
This comment block is confusing.  Can we just remove the code for now and resurrect it later if need be?